src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -346,15 +362,20 @@ private static IEnumerable<T> CollectToList<T>(IEnumerable<T> list, T element)
         /// specification and raise specification changed event.</param>
         /// <param name="elements">List of updated elements</param>
         private static void NotifySearchModelUpdate(ILibraryViewCustomization customization, IEnumerable<NodeSearchElement> elements)
-        {
-            var includes = elements
-                .Select(NodeItemDataProvider.GetFullyQualifiedName)
-                .Select(name => name.Split('.').First())
-                .Distinct()
-                .SkipWhile(s => s.Contains("://"))
-                .Select(p => new LayoutIncludeInfo() { path = p });
-
-            customization.AddIncludeInfo(includes, "Add-ons");
+        {  
+            //elements might be null if we have removed an element.
+            if (elements != null)
```

Comment:
this fixes a bug- whenever you change a custom node's info (so that it got removed) previously we were throwing once for every loaded custom node... this could be hundreds of exceptions.